tools/indexers/trials.js

```diff
@@ -332,11 +332,16 @@ const trialMapping = {
     },
     discrepancies: {
       properties: {
-        public_title: getDiscrepancyRecordMapping({
```

Comment:
Found [my answer](https://github.com/opentrials/api/commit/cc964c3b7180460ff4499dbd75cad823309b033d#diff-7b1832e030ab3a88fe949f72ada33c0bL114) and it appears the index attributes were missed when the model attributes were updated. So, this is safe to merge.